linera-core/src/remote_node.rs

```diff
@@ -327,23 +289,18 @@ impl<N: ValidatorNode> RemoteNode<N> {
         certificate: &GenericCertificate<T>,
         blob_ids: &[BlobId],
     ) -> Result<(), NodeError> {
-        // Find the missing blobs locally and retry.
+        ensure!(!blob_ids.is_empty(), NodeError::EmptyBlobsNotFound);
```

Comment:
Why is that an error if `blob_ids` is empty? Why not return `Ok(())`?

Reply:
The remote node shouldn't return a "missing blobs" error if it's not missing any blobs.

Reply:
No, no inlining please. Our methods are already hard to read with all the `self.a.b.c.d.e.foo()` stuff.